src_assets/windows/assets/shaders/directx/include/base_vs.hlsl

```diff
@@ -25,7 +25,7 @@ vertex_t generate_fullscreen_triangle_vertex(uint vertex_id, float2 subsample_of
         float2 rotation_center = { 0.5, 0.5 };
         tex_coord = round(rotation_center + mul(rotation_matrix, tex_coord - rotation_center));
 

```

Comment:
```suggestion

        // Swap the xy offset coordinates if the texture is rotated an odd number of times.
```

Reply:
Myself and a number of others from the logs I have seen received these warnings during sunshine startup during shader compilation:

#####
Info: Compiling shaders...
Warning: include/base_vs.hlsl(28,13-36): warning X3556: integer modulus may be much slower, try using uints if possible.
Warning: include/base_vs.hlsl(28,13-36): warning X3556: integer modulus may be much slower, try using uints if possible.
Info: System tray created
Warning: include/base_vs.hlsl(28,13-36): warning X3556: integer modulus may be much slower, try using uints if possible.
Warning: include/base_vs.hlsl(28,13-36): warning X3556: integer modulus may be much slower, try using uints if possible.
Warning: include/base_vs.hlsl(28,13-36): warning X3556: integer modulus may be much slower, try using uints if possible.
Warning: include/base_vs.hlsl(28,13-36): warning X3556: integer modulus may be much slower, try using uints if possible.
Info: Compiled shaders
####

This is due to the fact most GPU hardware (Nvidia and at least some Intel GPUs are less efficient at performing division operations and do not compile the source code for integers or float values to optimal lower level code. I'm not sure if the difference will be noticeable other than decluttering the logs but it's a 2 character change that even if it's not technically necessary, is an improvement.

Reply:
Oh, I'm fine with this and I have seen the log myself.

My only suggestion is to add a comment explaining that this is a quick "is odd" check for people that are unfamiliar with bitwise operations.

Reply:
Ah yeh sorry it's not exactly self explanatory 😅, I've added a comment for future reference

Reply:
@iMakeSoftware I think there might be a miscommunication somewhere. The suggestion is to add a comment to the code, not to the PR.

Reply:
No, not a miscommunication the second time... just a horrible UI in the GitHub mobile app, please see latest commit with comment added on line 27 as per suggestion